### DISCUSSION TITLE
Strategy Triple (Multi)

### DISCUSSION THREAD

**Discussion by kenorb** [EA31337/EA31337-strategies-meta#61] [Strategy Ideas]
Similar to EA31337/EA31337-strategies#30, it's strategy to run three strategies at the same time based on some priority criteria.

**Comment by kenorb** [comment]
Implemented [Multi meta strategy](https://github.com/EA31337/Strategy-Meta_Multi).

**Comment by mavidelisi** [comment]
Hi Kenorby ;

i add strategy to EA source and compiled but i cant add to ea parameter.how do you add strategy to EA .

i readed "how to add strategy manual" but this strategy has no indicator. it is multi strategy. i cant add. Please help me. How can i implement multi meta strategy and where will i change code from source code ?

**Reply by kenorb** [reply]
This multi strategy is added to v3.000-dev branch of the EA source code, unless you're talking about the different one. If you mean to see more parameters, check `config/define.h` file of Meta_Multi strategy, you'll see more when you uncomment `__input__`. If you mean different strategy or something else, create a new issue with the details.

**Reply by mavidelisi** [reply]
Ok. Thanks . i will test

**Reply by mavidelisi** [reply]
I have implemented and compiled Multi, Mirror and meta reversal strategies in both master and v3.000-dev source code. I also changed the parameters of the strategies, it didn't work at all and not open position. In the same way, I also cloned the v3.000-dev branch and compiled it again. I worked with the default parameters and no position was opened. I also changed the parameters. As an example, I activated the strategy (1<<0) for the strategy ATR. He has never opened a position. Could there be a bug in the code? 
Can you check it again?  If you have been able to open a position, can you send a parameter sample?
Thank you.

**Reply by kenorb** [reply]
Yes, it could be a bug, as I haven't tested properly these meta strategies yet.
